eiffel-vocabulary/EiffelActivityFinishedEvent.md

```diff
@@ -221,8 +221,8 @@ __Description:__ The number of the event within the named sequence.
 ## Version History
 | Version   | Introduced in                                          | Changes                                 |
 | --------- | ------------------------------------------------------ | --------------------------------------- |
-| 3.2.0     | Current version                                        | Add links.domainId member (see [Issue 233](https://github.com/eiffel-community/eiffel/issues/233)). |
-| 3.1.0     | Current version                                        | Add `data.persistentLogs.{mediaType,tags}`.   |
+| 3.2.0     | [edition-lyon](../../../tree/edition-lyon)             | Add links.domainId member (see [Issue 233](https://github.com/eiffel-community/eiffel/issues/233)). |
+| 3.1.0     | [edition-lyon](../../../tree/edition-lyon)             | Add `data.persistentLogs.{mediaType,tags}`.   |
```

Comment:
Should we really say that 3.1.0 was introduced in edition-lyon? Or should we instead just have a commit id here as it is done for 2.0.0? As edition Lyon contains 3.2.0.

Reply:
```suggestion
| 3.1.0     | No edition set            | Add `data.persistentLogs.{mediaType,tags}`.   |
```

Reply:
Emil and I talked about this and it's not clear what the Version history table is supposed to convey. Listing both 3.1.0 and 3.2.0 (as in this example) as introduced in Lyon is reasonable in the sense that the changes in both versions were first introduced in the Lyon edition, but if we interpret the table as something that focuses on the event versions it's only 3.2.0 that was introduced in Lyon. 3.1.0 is an "orphan" version that nobody is expected to be using (although it's technically fine to use it).

I'm fine with just about anything right now but I'd like us to figure out something that makes sense and (post-Lyon) align all the Version history tables accordingly.